tensorflow-serving-cluster/tensorflow-serving/docker/build_graphene_tf_serving_image.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-3.0-or-later
+# Copyright (C) 2021 Intel Corporation. Yunge Zhu <yunge.zhu@intel.linux.com>
+
+set -e
+
+if  [ ! -n "$1" ] ; then
+    tag=latest
+else
+    tag=$1
+fi
+
+# You can remove build-arg http_proxy and https_proxy if your network doesn't need it
+proxy_server="" # your http proxy server
+
+DOCKER_BUILDKIT=0 docker build \
+    -f graphene_tf_serving.dockerfile . \
+    -t graphene_tf_serving:${tag} \
+    --build-arg http_proxy=${proxy_server} \
+    --build-arg https_proxy=${proxy_server} \
+    --build-arg no_proxy=localhost,127.0.0.0/1 \
```

Comment:
use "    --build-arg no_proxy=localhost,127.0.0.0/1" instead of "    --build-arg no_proxy=localhost,127.0.0.0/1 \"